### PR TITLE
Add jax and jaxlib mapping

### PR DESCRIPTION
**Pull request by halejo-as**
Please add the following dependency to the rosdep database.

## Jax and JaxLib:

`jax`
`jaxlib`

## Package Upstream Source:

[jax](https://github.com/google/jax)

## Purpose of using this:

What’s new is that JAX uses [XLA](https://www.tensorflow.org/xla) to compile and run your NumPy programs on GPUs and TPUs.

## Links to Distribution Packages

- Debian:
  https://pypi.org/project/jax/
  https://pypi.org/project/jaxlib/
- Ubuntu: 
  https://pypi.org/project/jax/
  https://pypi.org/project/jaxlib/

### CODE REVIEW

**Review by clalancette — CHANGES_REQUESTED** [review]
Please see our [contributing guidelines for pip](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#pip-1), where both of these keys should be suffixed with `-pip`.

**Comment by ivanpauno** [comment]
@halejo-as friendly ping

File: rosdep/python.yaml [hunk 1]
```diff
@@ -7122,6 +7122,14 @@ python3-interpreter:
   rhel: [python3]
   slackware: [python3]
   ubuntu: [python3-minimal]
+python3-jax:
```
Comment (methylDragon) [inline rosdep/python.yaml:7125]:
```suggestion
python3-jax-pip:
```

File: rosdep/python.yaml [hunk 2]
```diff
@@ -7122,6 +7122,14 @@ python3-interpreter:
...
+  ubuntu:
+    pip:
+      packages: [jax]
+python3-jaxlib:
```
Comment (methylDragon) [inline rosdep/python.yaml:7129]:
```suggestion
python3-jaxlib-pip:
```

**Review by methylDragon — CHANGES_REQUESTED** [review]
:>

**Comment by halejo-as** [comment]
Thanks @methylDragon for the changes

**Comment by nuclearsandwich** [comment]
@halejo-as it looks like the current outstanding issue is that the entry is out of order.

```
ERR: list out of alphabetical order line 7200.  'python3-j1939-pip' should come before 'python3-jaxlib'
```

**Comment by ivanpauno** [comment]
It has happened more than one month since the last comment, so closing.
@halejo-as Please reopen or open a new PR when it's ready.
Thanks!

**Pull request closed without merging**